pubspec.yaml

```diff
@@ -5,9 +5,10 @@ description: Utilities for diffing two JSON objects
 homepage: https://github.com/google/dart-json_diff
 
 environment:
-  sdk: '>=2.2.0 <3.0.0'
+  sdk: '>=2.8.4 <3.0.0'
```

Comment:
Why does this code require 2.8.4?

Reply:
You're right, technically the requirements are 2.3.0 for the collection spreads